preset/deep-learning/Dockerfile

```diff
@@ -172,7 +172,7 @@ ARG TORCHAUDIO_CPU_VERSION
 ARG DEEPSPEED_VERSION
 
 # PyTorch CPU Env - conda packages
-RUN conda create -yn pytorch-cpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda create -yn pytorch-cpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```

Comment:
`intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-cpu -c "${INTEL_CHANNEL}" -c conda-forge \`